org.moreunit.core/src/org/moreunit/core/preferences/FolderPatternGroup.java

```diff
@@ -0,0 +1,190 @@
+package org.moreunit.core.preferences;
+
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.ModifyEvent;
+import org.eclipse.swt.events.ModifyListener;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.swt.widgets.Text;
+import org.eclipse.ui.forms.widgets.ExpandableComposite;
+import org.moreunit.core.matching.TestFolderPathPattern;
+import org.moreunit.core.ui.Composites;
+import org.moreunit.core.ui.ExpandableCompositeContainer;
+import org.moreunit.core.ui.Labels;
+import org.moreunit.core.ui.LayoutData;
+import org.moreunit.core.ui.ExpandableCompositeContainer.ExpandableContent;
+
+public class FolderPatternGroup implements GenericPreferencesGroup
+{
+    private static final int EXPLANATION_WIDTH_HINT = 350;
+    
+    private final ExpandableCompositeContainer container;
+    private final LanguagePreferencesWriter prefWriter;
+    private final Composite folderTplGroup;
+    
+    private Text srcFolderTemplateField;
+    private Text testFolderTemplateField;
+    
+    public FolderPatternGroup(Composite parent, ExpandableCompositeContainer container, LanguagePreferencesWriter prefsWriter)
+    {
+        this.container = container;
+        this.prefWriter = prefsWriter;
+
+        this.folderTplGroup = Composites.gridGroup(parent, "Rule for locating test files:", 2, 10);
+    }
+    
+    @Override
+    public void createContents()
+    {
+        createFolderTemplateFields(folderTplGroup);
+        createFolderTplExplanations(folderTplGroup);
+        
+    }
+
+    @Override
+    public void forceFocus()
+    {
+        // TODO Auto-generated method stub

```

Comment:
Can you remove the TODO please?
If nothing, to put inside worth specifying a comment "// no action" or something like that